### PR TITLE
Fix Preparations being in Spanish

### PR DESCRIPTION
**Pull request by grantfitzsimmons**
Solves #3684

### CODE REVIEW

File: specifyweb/frontend/js_src/lib/localization/stats.tsx [hunk 1]
```diff
@@ -13,232 +13,299 @@ export const statsText = createDictionary({
...
   collectionObjects: {
     'en-us': 'Collection Objects',
     'ru-ru': 'Объекты коллекции',
+    'es-es': 'Objetos de la Colección',
   },
   preparations: {
-    'en-us': 'Preparaciones',
+    'en-us': 'Preparations',
+    'en-es': 'Preparaciones',
+    'ru-ru': 'Подготовки',
```
Comment (maxxxxxdlp) [inline specifyweb/frontend/js_src/lib/localization/stats.tsx:31]: looks good

**Review by maxxxxxdlp — APPROVED** [review]
who did all those Spanish localizations? google translate?

remember that weblate will automatically run google translate and add localizations as soon as code is merged into `production`

**Comment by grantfitzsimmons** [comment]
> remember that weblate will automatically run google translate and add localizations as soon as code is merged into `production`

I just wanted to give Raycast a chance to translate :wink:

**Comment by maxxxxxdlp** [comment]
> > remember that weblate will automatically run google translate and add localizations as soon as code is merged into `production`
> 
> I just wanted to give Raycast a chance to translate 😉

good idea 😊 letting it work it's monthly pay

**Pull request merged**